.github/workflows/pull_request_build.yml

```diff
@@ -41,15 +41,15 @@ jobs:
       run: bash ./modules/p2/prepareDependenciesForP2.sh
 
     - name: Build with Maven
-      run: mvn -B clean install -Dorg.slf4j.simpleLogger.log.org.apache.maven.cli.transfer.Slf4jMavenTransferListener=warn
+      run: mvn -B clean install -Pcoverage -Dorg.slf4j.simpleLogger.log.org.apache.maven.cli.transfer.Slf4jMavenTransferListener=warn
```

Comment:
The commit message should mention in what context SonarCloud is enabled (in the context of GitHub actions), and what's the benefit of doing so (static code analysis).

Reply:
Some further commit message nits to address *optionally*:
- "Enable the SonarCloud step in the GH action script" should simply become the title.
- For the common reader it's unclear what "With GH actions this is possible for PR builds as well" refers to, i.e. what the problem with Travis (not PRs in general) is. A link to https://docs.travis-ci.com/user/environment-variables/#defining-encrypted-variables-in-travisyml would help to explain that.